src/Resources/contao/config/default.php

```diff
@@ -159,4 +159,3 @@
 $GLOBALS['TL_CONFIG']['proxyServerIps']       = '';
 $GLOBALS['TL_CONFIG']['sslProxyDomain']       = '';
 $GLOBALS['TL_CONFIG']['maintenanceMode']      = true;
-$GLOBALS['TL_CONFIG']['errorReporting']       = E_ALL & ~E_NOTICE & ~E_STRICT & ~E_DEPRECATED;
```

Comment:
As long as you are ok with the fact that the users cause the system to not boot anymore I can re add it.